content/en/logs/guide/increase-number-of-log-files-tailed.md

```diff
@@ -15,9 +15,7 @@ further_reading:
   text: "How to investigate a log parsing issue?"
 ---
 
-By default the Agent can tail up to 200 log files on Windows and MacOS, and 500 log files on other operating systems. This limit is set to avoid performances issue when wildcards are set on huge directories.
-
-To increase this limit, set the value of `open_files_limit` in the Agent's configuration file (`/etc/datadog-agent/datadog.yaml`) in the `logs_config` section:
+The `logs_config.open_files_limit` parameter in the Agent's configuration file (`/etc/datadog-agent/datadog.yaml`) determines the maximum number of log files the Agent can tail simultaneously. This limit is set to prevent performance issues when wildcards are set on huge directories. You can increase this limit by adjusting the parameter.
```

Comment:
```suggestion
The `logs_config.open_files_limit` parameter in the Agent's configuration file (`/etc/datadog-agent/datadog.yaml`) determines the maximum number of log files the Agent can tail simultaneously. This limit is set to prevent performance issues when wildcards are set on huge directories. You can increase the limit by adjusting this parameter.
```